sycamore/tests/unit/transforms/test_partition.py

```diff
@@ -122,3 +127,13 @@ def test_partition_html(self, mocker, path, partition_count) -> None:
         docset = partition.execute()
         doc = docset.take(limit=1)[0]
         assert len(doc["elements"]["array"]) == partition_count
+
+    @pytest.mark.parametrize("path, partition_count", [(TEST_DIR / "resources/data/pptx/design.pptx", 71)])
+    def test_partition_pptx(self, mocker, path, partition_count) -> None:
+        scan = mocker.Mock(spec=BinaryScan)
```

Comment:
Are we actually executing the partition logic here? What is the mocking doing for us?

Reply:
Yes, we are. The following two lines are setting up the mock behavior :
```
execute: Callable[[], Dataset] = _make_scan_executor(path, "pptx")
mocker.patch.object(scan, "execute", execute)
```